src/psyclone/tests/conftest.py

```diff
@@ -115,12 +114,11 @@ def setup_config_before_constants():
     is created before the config file was read: since some of the values
     of LFRicConstants depend on the config file, we have to make sure that
     a user-specified config file is read before creating an instance of
-    LFRicConstants. This flag is set from the various main entry points
-    of psyclone (generator.py, kernel_tools.py, psyad/main.py ). But the
+    LFRicConstants. This flag is set when `Config.load()` is called. But the
     tests will not set this flag. This fixture will make sure that
```

Comment:
Presumably there are some tests where a Config file is read? Is it safe to just turn this off for all tests?

Reply:
Actually, I see this is covered in `lfric_constants_test.py` so I'm happy.

Reply:
Yes, `lfric_constants_test`, but also `generator_test,test_main_api` (to make sure it works when calling `main`),